python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2181,15 +2183,31 @@ def _repr(self):
 # return tuple(index)
 
     def _subjobs_proxy(self):
+
         from Ganga.GPIDev.Lib.Registry.JobRegistry import JobRegistrySlice, _wrap
-        subjobs = JobRegistrySlice('jobs(%s).subjobs' % str(self.id))
-        if isType(self.subjobs, SubJobXMLList):
-            subjobs.objects = self.subjobs
-        else:
-            for sj in self.subjobs:
-                subjobs.objects[sj.id] = sj
-        #print('return slice: %s' % str(subjobs))
-        return _wrap(subjobs)
+        from Ganga.GPIDev.Base.Proxy import stripProxy
+        if not isType(self._stored_subjobs_proxy, JobRegistrySlice):
+            subjob_slice = JobRegistrySlice('jobs(%s).subjobs' % str(self.id))
+            self._stored_subjobs_proxy = _wrap(subjob_slice)
```

Comment:
I thought that *did* work. We made the proxy system so that it stores the proxy-wrapper of a `GangaObject` in an attribute on it so that doing `addProxy` multiple times to an object will always return the same proxy object.
```python
j = LocalFile()  # A non-proxied object
assert addProxy(j) is addProxy(j)
```
?